tools/Makefile

```diff
@@ -64,20 +48,19 @@ tools-y += sstrip
 tools-y += zip
 tools-y += zlib
 tools-y += zstd
-tools-$(if $(CONFIG_BUILD_ALL_HOST_TOOLS),y) += liblzo
-tools-$(if $(CONFIG_BUILD_ALL_HOST_TOOLS)$(BUILD_B43_TOOLS),y) += b43-tools
-tools-$(if $(CONFIG_BUILD_ALL_HOST_TOOLS)$(BUILD_ISL),y) += isl
-tools-$(if $(CONFIG_BUILD_ALL_HOST_TOOLS)$(BUILD_LZ4_TOOLS),y) += lz4
-tools-$(if $(CONFIG_BUILD_ALL_HOST_TOOLS)$(BUILD_LZO_TOOLS),y) += lzop
-tools-$(if $(CONFIG_BUILD_ALL_HOST_TOOLS)$(BUILD_TOOLCHAIN),y) += gmp mpc mpfr
```

Comment:
`mpfr` and `gmp` are not added again.

Reply:
They get pulled in via the depweaving below [L94](https://github.com/openwrt/openwrt/blob/59b266c31b742b75a1bfbe503be66f5599ce2017/tools/Makefile#L94), having them up here seems useless.